components/nimbus/ios/Nimbus/NimbusApi.swift

```diff
@@ -39,6 +39,12 @@ public protocol NimbusFeatureConfiguration {
     func getVariables(featureId: String, recordExposureEvent: Bool) -> Variables
 }
 
+public extension NimbusFeatureConfiguration {
+    func getVariables(featureId: String) -> Variables {
+        return getVariables(featureId: featureId, recordExposureEvent: true)
+    }
+}
```

Comment:
Added this extension method because protocols don't allow default arguments.

Reply:
The implementation of this in Nimbus.swift also has a default value, I don't think we need it in both places if you are going to define this as an extension here (nifty solution, by the way).